debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/processor/infinispan/AbstractInfinispanLogMinerEventProcessor.java

```diff
@@ -356,6 +359,33 @@ protected Scn calculateNewStartScn(Scn endScn, Scn maxCommittedScn) throws Inter
         }
     }
 
+    /**
+     * Purge the necessary caches with all entries that occurred prior to the specified change number.
+     *
+     * @param minCacheScn the minimum system change number to keep entries until
+     */
+    protected abstract void purgeCache(Scn minCacheScn);
```

Comment:
@Naros It seems to me that both implementations of the method are the same, why does it need to be introduced as an abstract one?